chia/full_node/mempool_manager.py

```diff
@@ -545,7 +545,7 @@ async def add_spend_bundle(
             return SpendBundleAddInfo(item.cost, MempoolInclusionStatus.PENDING, [], err)
         else:
             # Cannot add to the mempool or pending pool.
-            return SpendBundleAddInfo(None, MempoolInclusionStatus.FAILED, [], err)
+            return SpendBundleAddInfo(uint64(0), MempoolInclusionStatus.FAILED, [], err)
```

Comment:
This is the unrelated change to `add_spend_bundle` that triggered several other changes in this patch (addressed already).